sdk/eventhub/event-hubs/README.md

```diff
@@ -37,30 +37,44 @@ You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your t
 
- For more concepts and deeper discussion, see: [Event Hubs Features](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features)
+For more concepts and deeper discussion, see: [Event Hubs Features](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features)
 
- 
 ### Authenticate the client
 
 Interaction with Event Hubs starts with an instance of the [EventHubClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/event-hubs/eventhubclient?view=azure-node-latest) class. You can instantiate
 this class using one of the below
 
-- [EventHubClient](link)
-  - This constructor takes the connection string of the form 'Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key;' and entity name to your Event Hub instance. You can get the connection string from the [Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal)
-- [EventHubClient](link)
-  - This constructor takes only the connection string of the form 'Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key;EntityPath=my-event-hub-name'. You can get the connection string from the [Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal)
-- [EventHubClient](link) 
-  - This constructor takes the host name and entity name of your Event Hub instance and credential that implements the TokenCredential interface. The host name is of the format `name-of-event-hub-instance.servicebus.windows.net`.
+```javascript
+const client = new EventHubClient("my-connection-string", "my-event-hub");
+```
+
+- This constructor takes the connection string of the form 'Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key;' and entity name to your Event Hub instance. You can get the connection string from the [Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal)
+
+```javascript
+const client = new EventHubClient("my-connection-string");
+```
+
+- This constructor takes only the connection string of the form 'Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key;EntityPath=my-event-hub-name'. You can get the connection string from the [Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal)
+
+If the connection string is copied from the Event Hubs namespace, it will likely not contain the path to the desired Event Hub, which is needed. In this case, the path can be added manually by adding ";EntityPath=[[ EVENT HUB NAME ]]" to the end of the connection string. For example, ";EntityPath=my-event-hub-name".
```

Comment:
The line before this is asking to copy from the portal which will definitely give connection string without entity path. And then we say "**If** the connection string is copied". It is weird to say `if` for something that we suggested to do just previously.

I would remove the previous line with the link to the Azure portal entirely and update this line here to

> The [connection string from Azure Portal](... link to Azure portal doc here) is for the entire Event Hubs namespace and will not contain the path to the desired Event Hub instance which is needed for this constructor overload. In this case, the path ...

Reply:
Updated!!